docs/guides/dataweavetransformation.md

```diff
@@ -0,0 +1,311 @@
+	</buyer>
+	<shop>main branch</shop>
+	<salesperson>Liam Smith</salesperson>
+</order>
+```
+
+Into this new JSON document:
+```json
+{
+  "email": "james@hotmail.com",
+  "name": "James"
+}
+```
+
+Let's go step by step to see how we can deploy this transformation as part of a TriggerMesh Bridge.
+
+Below is a diagram of the Bridge we will construct.
+
+![](../assets/images/dataweavetransformation.png)
+
+## Deploy the Broker
+Deploy a Broker by writing the following YAML in a file:
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: demo
+```
+
+Create the Broker with the following command:
+```console
+kubectl apply -f <manifest.yaml>
+```
+
+## Deploying the `EventDisplay` Service
+Let's now deploy the end of our event flow. The `EventDisplay` is a simple application that can be used to display CloudEvents. It can 
+be deployed by writing the following YAML in a file and using `kubectl apply -f <manifest.yaml>`:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+    name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+```
+
+## Deploy the `DataWeaveTransformation` Object
+With the `event-display` in place, the `DataWeaveTransformation` object can now be deployed in the same manner using the following manifest. It contains an inline DataWeave spell that will be used by default but can be overridden by passing a spell in the CloudEvent payload.
+
+```yaml
+apiVersion: flow.triggermesh.io/v1alpha1
+kind: DataWeaveTransformation
+metadata:
+  name: demo
+spec:
+  allowPerEventDwSpell: true
+  dwSpell:
+    value: |-
+      %dw 2.0
+      output application/json
+      ---
+      {
+          email: payload.order.buyer.email,
+          name: payload.order.buyer.name,
+      }
+  inputContentType: application/xml
+  outputContentType: application/json
+```
+
+## Configure the Triggers
+Next, Triggers need to be configured to route our Cloudevents to the `DataWeaveTransformation` and `EventDisplay` objects. This can be done by writing the following YAML in a file and using `kubectl apply -f <manifest.yaml>`. We have two triggers, one to send events containing XML to the transformation and one to send all events to the event display.
+
+```yaml
+kind: Trigger
+apiVersion: eventing.knative.dev/v1
+metadata:
+  name: event-display
+spec:
+  broker: demo
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+---
+kind: Trigger
+apiVersion: eventing.knative.dev/v1
+metadata:
+  name: dataweavetransformation-xmldoc
+spec:
+  broker: demo
+  filter:
+    attributes:
+      # setting a filter to process only events of type `xml.document`
+      type: xml.document
+  subscriber:
+    ref:
+      apiVersion: flow.triggermesh.io/v1alpha1
+      kind: DataWeaveTransformation
+      name: demo
+```
+
+## Deploy a Curl Pod
```

Comment:
should we use a define a `WebhookSource` instead?

Reply:
@sameersbn I was following the same xslt docs structure to have something similar.